From 8f78a8014c6394aa3653d2e1519c62ac8849354e Mon Sep 17 00:00:00 2001
From: Yi Sun <yi.sun@intel.com>
Date: Mon, 22 Feb 2016 14:22:11 +0800
Subject: [PATCH 366/403] Compile i915/vgt/* with Werror.

Filter vgt/*, and add compile option -Werror, which will stop code
compiling if any warning existed.

Signed-off-by: Yi Sun <yi.sun@intel.com>
Signed-off-by: Zhu, Libo <libo.zhu@intel.com>
---
 drivers/gpu/drm/i915/Makefile |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/gpu/drm/i915/Makefile b/drivers/gpu/drm/i915/Makefile
index 5dd2d22..ffa0532 100644
--- a/drivers/gpu/drm/i915/Makefile
+++ b/drivers/gpu/drm/i915/Makefile
@@ -128,3 +128,6 @@ i915-$(CONFIG_I915_VGT)  += $(VGT)/vgt.o	\
 			   $(VGT)/legacy/render_hsw.o
 
 CFLAGS_i915_trace_points.o := -I$(src)
+
+$(foreach fname, $(filter $(VGT)/%, $(i915-y)), $(src)/$(fname)):\
+	ccflags-$(CONFIG_I915_VGT) += -Werror
-- 
1.7.10.4

